### PR TITLE
Example to configure GitHub Authentication

### PR DESCRIPTION
**Pull request by ch-stark**
Tested with a 4.5 Spoke Cluster

### CODE REVIEW

File: community/CM-Configuration-Management/policy-github-oauth-sample.yaml [hunk 1]
```diff
@@ -0,0 +1,96 @@
+apiVersion: policy.open-cluster-management.io/v1
+kind: Policy
+metadata:
+  name: policy-github-oauth
+  namespace: default
```
Comment (ycao56) [inline community/CM-Configuration-Management/policy-github-oauth-sample.yaml:5]: remove ns

File: community/CM-Configuration-Management/policy-github-oauth-sample.yaml [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+                  namespace: openshift-config
+                type: Opaque
+                data:
+                  clientSecret: PUTSECRETHERE
+---
+apiVersion: policy.open-cluster-management.io/v1
+kind: PlacementBinding
+metadata:
+  name: binding-policy-github-oauth
+  namespace: default
```
Comment (ycao56) [inline community/CM-Configuration-Management/policy-github-oauth-sample.yaml:74]: remove ns

File: community/CM-Configuration-Management/policy-github-oauth-sample.yaml [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+subjects:
+- name: policy-github-oauth
+  kind: Policy
+  apiGroup: policy.open-cluster-management.io
+---
+apiVersion: apps.open-cluster-management.io/v1
+kind: PlacementRule
+metadata:
+  name: placement-policy-github-oauth
+  namespace: default
```
Comment (ycao56) [inline community/CM-Configuration-Management/policy-github-oauth-sample.yaml:88]: remove ns

**Review by dockerymick — APPROVED** [review]
Made an update to the description

**Comment by ycao56** [comment]
can you resolve the conflicts? we updated the table to use markdown table instead

**Comment by dockerymick** [comment]
Conflict fixed in the table, continuing to merge.

**Pull request merged**